samba/config.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-version: 11.0.0
+version: 11.1.0
```

Comment:
```suggestion
version: 11.0.0
```

Reply:
Adding this one, to avoid the upgrade/release. Got another PR inbound for the 11.1.0